src/cljc/ataru/schema/form_schema.cljc

```diff
@@ -24,13 +24,14 @@
 ;         `  `                     "`_,-','/       ,-'"  /
 ;                                 ,'",__,-'       /,, ,-'
 ;                                 Vvv'            VVv'
-(s/defschema Form {(s/optional-key :id)           s/Int
-                   :name                          s/Str
-                   (s/optional-key :key)          s/Str
-                   (s/optional-key :created-by)   s/Str
-                   (s/optional-key :created-time) #?(:clj  org.joda.time.DateTime
-                                                     :cljs s/Str)
-                   s/Any                          s/Any})
+(s/defschema Form {(s/optional-key :id)                            s/Int
+                   :name                                           s/Str
+                   (s/optional-key :key)                           s/Str
+                   (s/optional-key :created-by)                    s/Str
+                   (s/optional-key :created-time)                  #?(:clj  org.joda.time.DateTime
+                                                                      :cljs s/Str)
+                   (s/optional-key :unprocessed-application-count) s/Int
```

Comment:
Just a thought (not really related to this PR): we should probably have some sort of form/application "metadata" definitions, since now we're injecting stuff like hakukohde/haku/application stuff when loading data via API that's not technically part of the "model" or present when uploading the same "models" to the API.